### PR TITLE
Adding cid

### PR DESCRIPTION
**Pull request by ttyerl**
Adding content_id if the Bamboo.Attachment struct has one

### CODE REVIEW

File: lib/bamboo/adapters/smtp_adapter.ex [hunk 1]
```diff
@@ -187,6 +187,16 @@ defmodule Bamboo.SMTPAdapter do
   end
 
   defp add_attachment_header(body, attachment) do
```
Comment (julien-leclercq) [inline lib/bamboo/adapters/smtp_adapter.ex]: isn't this clause always matching ?

File: lib/bamboo/adapters/smtp_adapter.ex [hunk 2]
```diff
@@ -192,6 +192,16 @@ defmodule Bamboo.SMTPAdapter do
...
+    case attachment.content_id do
```
Comment (kdisneur) [inline lib/bamboo/adapters/smtp_adapter.ex:195]: Looking at [`Bamboo.Attachment`](https://github.com/thoughtbot/bamboo/blob/master/lib/bamboo/attachment.ex), I don't understand where the key `content_id` is defined?

**Comment by PhilWaldmann** [comment]
any updates?

**Comment by arnaudmorisset** [comment]
Hello @ttyerl and @PhilWaldmann 👋 

We are sorry for the delay. 🙇‍♂️

We didn't have the time to manage this project properly, but it should be better now.

The feature you asked for is available in our latest release: https://github.com/fewlinesco/bamboo_smtp/releases/tag/v3.1.0

**Pull request closed without merging**